R/RcppLdpath.R

```diff
@@ -47,12 +47,16 @@ canUseCXX0X <- function() { TRUE }  # .Call( "canUseCXX0X", PACKAGE = "Rcpp" )
 
 ## Provide compiler flags -- i.e. -I/path/to/Rcpp.h
 RcppCxxFlags <- function(cxx0x=FALSE) {
-    # path <- RcppLdPath()
+
     path <- Rcpp.system.file( "include" )
     if (.Platform$OS.type=="windows") {
         path <- asBuildPath(path)				# #nocov
     }
-    paste("-I", path, if (cxx0x && canUseCXX0X()) " -std=c++0x" else "", sep="")
+
+    paste0(
```

Comment:
Can you make that line, please, to remain consistent with the rest of the code of the package?

Reply:
Updated / simplified.